platform/darwin/src/MGLAccountManager.m

```diff
@@ -104,7 +104,8 @@ + (NSString *)skuToken {
         return (NSString *)[mbx valueForKeyPath:MGLAccountManagerExternalMethodName];
     }
     
-    return MBXSKUToken.skuToken;
+    //return MBXSKUToken.skuToken;
```

Comment:
Re-enable sku tokens.
There’s a static lib and header at `platform/ios/vendor/mapbox-accounts-ios/`